packages/codegen-ui/lib/types/form/form-cta.ts

```diff
@@ -0,0 +1,42 @@
+/*
+  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+  Licensed under the Apache License, Version 2.0 (the "License").
+  You may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+ */
+type VisibleStudioFormButton = {
+  visible: true;
+
+  labelOverride?: string;
+};
+
+type HiddenStudioFormButton = {
+  visible: false;
+};
+
+type StudioFormButton = VisibleStudioFormButton | HiddenStudioFormButton;
+
+/**
+ * Configuration for each of the specified CTA's
+ */
+export type StudioFormButtons = {
+  /**
+   * The position of the CTA's in the form when rendered
+   */
+  position?: 'Top' | 'Bottom' | 'TopAndBottom';
```

Comment:
is this the standard casing we're using?

Reply:
updated to use camelCase